apps/op-func/src/infra/storage/access-log-register.ts

```diff
@@ -40,9 +28,15 @@ export class AccessLogRegister {
     // get downloaded data by accessing downloadBlockBlobResponse.readableStreamBody
     const downloadBlockBlobResponse = await blobClient.download();
```

Comment:
is it safe? maybe it should be inside the `try` block

Reply:
It is not safe. I put everything inside the `try` block because also the `getBlobClient` method can fail